content/applications/hr/appraisals/appraisal_analysis.rst

```diff
@@ -0,0 +1,109 @@
+==================
+Appraisal analysis
+==================
+
+The **Appraisals** app has the ability to report on all the appraisals in the system, including
+past, present, and future appraisals, and their respective statuses. This report can assist managers
+to see when the various appraisals are scheduled, and see if any appraisals are overdue or have not
+been confirmed yet.
+
+To access the *Appraisal Analysis* report, navigate to :menuselection:`Appraisals app --> Reporting
+--> Appraisal Analysis`.
+
+On the :guilabel:`Appraisal Analysis` page, a report loads, displaying all the appraisals in the
+database. Each entry is highlighted in different color to represent their status:
+
++-----------+----------------------------+-----------------------------------------------+
+| Color     | Status                     | Meaning                                       |
++===========+============================+===============================================+
+| Yellow    |:guilabel:`Done`            | The appraisal was completed.                  |
++-----------+----------------------------+-----------------------------------------------+
+| Orange    |:guilabel:`Appraisal Sent`  |The appraisal was confirmed but not completed. |
++-----------+----------------------------+-----------------------------------------------+
+| Red       |:guilabel:`Cancelled`       |The appraisal was cancelled.                   |
++-----------+----------------------------+-----------------------------------------------+
+| Gray      |:guilabel:`To Start`        | The appraisal was scheduled but not confirmed.|
++-----------+----------------------------+-----------------------------------------------+
```

Comment:
This table is a really good way to organize the info (:

Reply:
I started with my usual bullet list but then I realized this would be more helpful, visually =)